src/codegen/llvm/codegen_llvm.cc

```diff
@@ -688,6 +688,12 @@ llvm::Value* CodeGenLLVM::CreateIntrinsic(const Call* op) {
       indices.push_back(i);
     }
     return builder_->CreateShuffleVector(v0, v1, indices);
+  } else if (op->is_intrinsic("broadcast16")) {
```

Comment:
You can directly use value.astype("int8x16") to convert a int8 type into int8x16

Reply:
Thanks, that cleaned up the code.